docs/topics/configmap-mission-intro.adoc

```diff
@@ -5,9 +5,13 @@ This mission shows you how to:
 * Use the configuration provided by the `ConfigMap` within an application.
 * Deploy changes to the `ConfigMap` configuration of running applications.
 
+.About Externalized Configuration
+It is important for an application's configuration to be externalized and separate from its code. This allows for the application's configuration to change as it moves through different environments while leaving the code unchanged. This also keeps sensitive or internal information out of your codebase and version control. Many languages and application servers provide environment variables to support externalizing an application's configuration. Microservices and Linux containers increase the complexity of this by adding pods, or groups of containers representing a deployment, and polyglot environments.
```

Comment:
Last sentence: pods are a Kubernetes concept, not microservices nor Linux containers.

Reply:
@Ladicek thanks,  will change that sentence to reference Kubernetes instead.